<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>214</width>
      <height>469</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>214</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>270</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Reach</string>
     </property>
    </widget>
    <widget class="QSlider" name="reachSlider">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>22</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Yaw</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="wheelsIPAddressTextBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="armLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Arm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Pitch</string>
     </property>
    </widget>
    <widget class="QSlider" name="wheelsSlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>22</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="wheelsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wheels</string>
     </property>
    </widget>
    <widget class="QSlider" name="yawSlider">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>300</y>
       <width>22</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Wheels</string>
     </property>
    </widget>
    <widget class="QSlider" name="pitchSlider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>22</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="armIPAddressTextBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameTextBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordTextBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </widget>
   <widget class="VlcWidgetVideo" name="wheelsVideo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>641</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="VlcWidgetVideo" name="armVideo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="controlsHelpMenu"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="controlsHelpMenu">
   <property name="text">
    <string>Controls</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>usernameTextBox</tabstop>
  <tabstop>passwordTextBox</tabstop>
  <tabstop>wheelsIPAddressTextBox</tabstop>
  <tabstop>armIPAddressTextBox</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>wheelsSlider</tabstop>
  <tabstop>yawSlider</tabstop>
  <tabstop>pitchSlider</tabstop>
  <tabstop>reachSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
